contracts/FIFOMarket.sol

```diff
@@ -519,10 +528,9 @@ contract FIFOMarket is
    */
   function reserveRemoval(uint256 removalId) external returns (bool) {
     address supplierAddress = removalId.supplierAddress();
-    require(
-      _activeSupply[supplierAddress].removeRemoval(removalId),
-      "Market: removal not in active supply"
-    ); // TODO (Gas Optimization): Use custom error
+    if (!_activeSupply[supplierAddress].removeRemoval(removalId)) {
+      revert RemovalNotInActiveSupply({removalId: removalId});
```

Comment:
https://github.com/nori-dot-eco/contracts/pull/280/files#r913286493

Reply:
isn't this actually redundant given [this block of code](https://github.com/nori-dot-eco/contracts/pull/280/files#diff-15c315f8423ca6a9233aa5946874fc34024b7a4c80f9a6cd5c102bfc3cfa35f8R58-R65)